data_prep/generate_climos.py

```diff
@@ -90,45 +61,43 @@ def create_climo_file(fp_in, fp_out, t_start, t_end, variable):
         'tasmax', 'pr'
     }
 
-    if variable not in supported_vars:
-        raise Exception("Unsupported variable: cant't yet process {}".format(variable))
-
-    # Allow different ops by variable? # op = 'sum' if variable == 'pr' else 'mean'
```

Comment:
The question is, "is our operation always mean"? You have removed the comment that suggests that perhaps we want to use "sum" for precipitation, so can I assume that you clarified with the scientists that we always want to use mean in this context?